app/views/handlers/confirmation_email.py

```diff
@@ -92,7 +98,7 @@ def is_enabled(schema: QuestionnaireSchema):
 
 
 @dataclass
-class ConfirmationEmailFulfilmentRequest(FulfilmentRequest):
+class ConfirmationEmailTask(FulfilmentRequest):
```

Comment:
The name of this should be reverted to `ConfirmationEmailFulfilmentRequest` as this is the fulfilment request that goes into the task.